commons/addons/cellar-vs-fsbuckets.md

```diff
@@ -16,7 +16,7 @@ You want to store files but you can't decide between FS Buckets and Cellar. This
 will compare them: usage, pros/cons, pricing. Get ready for the ultimate comparison.
 
 Clever Cloud uses immutable throwable VMs.
-Every time you redeploy your application, you lose the old VMs and all the files stored on their filesystem.
+Every time you redeploy your application, you lose the old VM and all the files stored on their filesystem.
```

Comment:
Actually, if you remove the "s", change "their" by "its".

Reply:
```suggestion
Every time you redeploy your application, you lose the old VMs and all the files stored on their filesystems.
```

Reply:
I actually changed to "instances".